Zend/tests/closure_041.phpt

```diff
@@ -53,9 +53,9 @@ $d = $nonstaticScoped->bindTo(null); $d(); echo "\n";
 $d->bindTo($d);
 
 echo "After binding, with same-class instance for the bound ones", "\n";
-$d = $staticUnscoped->bindTo(new A); $d(); echo "\n";
+$d = $staticUnscoped->bindTo(new A); /* $d(); */ echo "\n";
```

Comment:
Right it does. The Closure::bind() functions are now returning NULL on failure. (Which also most functions do) … This is necessary, as we'd end up with badly bound Closures in case the scope doesn't fit otherwise.